### DISCUSSION TITLE
In poker, how many aces are there in a royal flush?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#702] [Galaxy Trivia] [answered]
**Bonus:** What is the next highest ranking poker hand after that?

**Comment by peterbaccaro** [comment]
In poker, there is only **one Ace** in a royal flush.  A royal flush is the highest-ranking hand and consists of the following cards: Ace, King, Queen, Jack, and Ten, all of the same suit.

The next highest ranking poker hand after a royal flush is a **straight flush**. A straight flush is any five cards of the same suit in sequence, such as 10, 9, 8, 7, 6 of spades.

**Comment by thuongtruong1009** [comment, marked as answer]
In poker, a royal flush is the highest-ranking hand and consists of a straight flush from 10 to Ace of the same suit. There are four aces in a standard deck of 52 cards, one of each suit.

The next highest ranking poker hand after a royal flush is a straight flush, which is any five cards of sequential rank, all of the same suit. The highest possible straight flush is a royal flush, but there are nine other possible straight flushes ranked in order from Ace to 10.
